src/explore-education-statistics-admin/src/pages/admin-dashboard/AdminDashboardPage.tsx

```diff
@@ -92,7 +92,19 @@ const AdminDashboardPage = ({ match }: RouteComponentProps<MatchProps>) => {
                   </li>
                 </span>
               </h1>
+
+              <p>This is your administration dashboard here you can:</p>
```

Comment:
Should this have a hyphen in it?

`This is your administration dashboard - here you can:`